apps/web/package.json

```diff
@@ -12,7 +12,7 @@
     "envsetup": "chmod +x ./env.sh && ./env.sh && mv env-config.js ./public/env-config.js",
     "start:static:build": "pnpm envsetup:docker && http-server build -p 4200 --proxy http://localhost:4200?",
     "start:docker": "pnpm build && pnpm start:static:build",
-    "start:dev": "pnpm start",
+    "start:dev": "cross-env NODE_OPTIONS=--max-old-space-size=4096 pnpm start",
```

Comment:
Any reason for updating this?

Reply:
Needed it for getting the web app running, without it, I usually get a JS heap memory limit error. Forgot to remove it after testing. Updated the code, should be fine now